build/source/org/zotero/integration/ooo/comp/Document.java

```diff
@@ -731,38 +739,48 @@ public XTextViewCursor getSelection() {
 	}
 	
 	private boolean checkForExportMarker() throws Exception {
-		XEnumerationAccess xParaAccess = UnoRuntime.queryInterface(
+		// get paragraphs
+		XEnumerationAccess paragraphs = UnoRuntime.queryInterface(
```

Comment:
I know variable name is not that straightforward, but this is the naming convention for UNO objects across the codebase, which differentiates them from our own object types, so let's maintain the consistency.